.pre-commit-config.yaml

```diff
@@ -2,8 +2,6 @@ exclude: |
   (?x)
   # NOT INSTALLABLE ADDONS
   # END NOT INSTALLABLE ADDONS
-  ^l10n_br_nfe_spec/models/v4_00/leiauteNFe.py|  # (don't reformat generated code)
-  ^spec_driven_model/tests/|  # (tests include generated code)
```

Comment:
@netosjb eu acho melhor manter essas duas linhas e não aplicar o pre-commit no código gerido. Isso vai gerir "noise" para nada e dificultar a checagem do código gerido se a gente for gerir de novo ou fazer o diff com alguma outra branch. Tou para substituir esse código por código gerido pelo xsdata, mas enfim até lá eu acho melhor evitar esse noise. Pode reverter essas duas linhas e o commit nos 2 módulos por favor?

Reply:
opa vou fixar isso